common/templates/emails/new_enrollment_email_subject.txt

```diff
@@ -0,0 +1 @@
+New student enrollment
```

Comment:
All text needs to be translated, e.g. https://github.com/edx/edx-platform/blob/master/common/templates/emails/sync_learner_profile_data_email_change_subject.txt

Reply:
And note that if you do this, you'll have to strip newlines. See https://github.com/edx/edx-platform/blob/master/common/djangoapps/third_party_auth/pipeline.py#L778-L782